test/map/transform.test.cpp

```diff
@@ -109,7 +109,13 @@ TEST(Transform, IntegerZoom) {
     auto checkIntegerZoom = [&transform](uint8_t zoomInt, double zoom) {
         double scale = transform.getState().zoomScale(zoom);
         transform.setScale(scale);
+#if defined(__ANDROID__)
```

Comment:
Can we instead use `#if __ANDROID__`? This define would also trigger in the case of `__ANDROID__ = 0`